ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -96,6 +94,100 @@ def test_basic_funnel_default_funnel_days(self):
         self.assertEqual(result[1]["count"], 1)
         self.assertEqual(len(result[1]["people"]), 1)
 
+    def test_basic_funnel_default_funnel_days_breakdown(self):
+        filters = {
+            "events": [
+                {
+                    "id": "user signed up",
+                    "type": "events",
+                    "order": 0,
+                    "properties": [
+                        {
+                            "key": "$current_url",
+                            "operator": "icontains",
+                            "type": "event",
+                            "value": "https://posthog.com/docs",
+                        }
+                    ],
+                },
+                {"id": "paid", "type": "events", "order": 1},
+            ],
+            "insight": INSIGHT_FUNNELS,
+            "date_from": "2020-01-01",
+            "date_to": "2020-01-14",
+            "breakdown": "$current_url",
+            "breakdown_type": "event",
+        }
+
+        filter = Filter(data=filters)
+        funnel = ClickhouseFunnel(filter, self.team)
+
+        # event
+        person = _create_person(distinct_ids=["user_1"], team_id=self.team.pk)
+        _create_event(
+            team=self.team,
+            event="user signed up",
+            distinct_id="user_1",
+            timestamp="2020-01-02T14:00:00Z",
+            properties={"$current_url": "https://posthog.com/docs/x"},
+        )
+        _create_event(
+            team=self.team,
+            event="paid",
+            distinct_id="user_1",
+            timestamp="2020-01-10T14:00:00Z",
+            properties={"$current_url": "https://posthog.com/docs/x"},
+        )
+
+        # dummy events to make sure that breakdown is not confused
+        # it was confused before due to the nature of fetching breakdown values with a LIMIT based on value popularity
+        # see https://github.com/PostHog/posthog/pull/5496
+        for current_url_letter in ascii_lowercase[:20]:
+            # twenty dummy breakdown values
+            for _ in range(2):
+                # each twice, so that the breakdown values from dummy events rank higher in raw order
+                # this test makes sure that events are prefiltered properly to avoid problems with this raw order
+                _create_event(
+                    team=self.team,
+                    event="user signed up",
+                    distinct_id="user_1",
+                    timestamp="2020-01-02T14:00:00Z",
+                    properties={"$current_url": f"https://posthog.com/blog/{current_url_letter}"},
+                )
+
+        with self.assertNumQueries(1):
```

Comment:
Note: This does not assert anything.

Reply:
It does assert _something_ I guess. I adapted the test above for this incl. this, so 🤷 .

Reply:
https://github.com/PostHog/posthog/blob/fix-element-empty-filtering/ee/clickhouse/util.py#L71-L77

Nope, it literally does nothing

It serves a purpose in django tests to guard against accidental N+1s from ORM but we don't really do that in clickhouse. :)